.github/workflows/ci-go.yml

```diff
@@ -50,3 +50,31 @@ jobs:
           go get .
           gosec -exclude=G204,G304,G107 -exclude-dir=protobuf/ ./...  
         working-directory: src
+
+  go-ginkgo-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Source
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-go@v2
+        with:
+          go-version: v1.18
+        
+      - name: Setup Enviroment
+        run: |  
+          # setup kubernetes environment
+          ./k3s/install_k3s.sh
+
+      - name: Run KubeArmor in background
+        run: |
+          make -C KubeArmor/ run &
```

Comment:
it will not work, `KubeArmor` source is not present here so we cannot build and run.

Reply:
resolved